### DISCUSSION TITLE
Please welcome our new moderators: Ben Clothier and Krool

### DISCUSSION THREAD

**Discussion by WaynePhillipsEA** [twinbasic/twinbasic#674] [General]
It is with great pleasure that I announce that @bclothier and @Kr00l have agreed to help me moderate the twinBASIC GitHub repository.  Please join me in welcoming them to their new positions of power!

Both are extremely trustworthy and sensible characters (unlike the rest of you lot 😁) and so they have my full backing to do whatever they see fit here.

# Welcome Ben and Krool!

**Comment by vbBaka** [comment]
I think it was a good decision. Both show to be diplomatic in their way of communication.
let's hope the power will not corrupt them.

**Comment by Greedquest** [comment]
Possibly worth putting in its own discussion/wiki, but some guidance on how best to use discussions might be useful (something to be moderated loosley).

Right now the main point I want to raise is on making the voting more useful. Something like this would be good:

---

On the bottom of posts you can leave a reaction - e.g. 
![image](https://user-images.githubusercontent.com/25348920/152983347-dd0b88d9-2745-4f9b-becd-df6d90009266.png)

 
- The up arrow indicates "This post is important"/ "This post should be seen by others" (but not necessarily that you agree with it)
- The thumb-up :+1: reaction indicates "I agree with this post" / "This post is good" / "This post is useful"
- The thumb-down :-1: reaction indicates "I don't agree with this post" / "This post is bad" / "This post is not useful"

Other reactions like 🎉or 👀 do not have any specific meaning but can be used as you see fit.

---

Formalising the reactions will make them more useful in filtering out important posts, as some discussions can get quite lengthy. Using the up-arrow as a neutral way to get eyes on a post is very valuable, we should do it more.
